.github/workflows/release-please.yml

```diff
@@ -0,0 +1,13 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
The default branch should be `master`?

Reply:
Good catch and pushing a tag.

Reply:
@jyeshe the tool will automatically tag and release